clang/lib/CodeGen/CGCall.cpp

```diff
@@ -2022,6 +2022,19 @@ static void getTrivialDefaultFunctionAttributes(
     std::tie(Var, Value) = Attr.split('=');
     FuncAttrs.addAttribute(Var, Value);
   }
+
+  TargetInfo::BranchProtectionInfo BPI(LangOpts);
+
```

Comment:
after #82819 lands, this can be 2 lines
```suggestion
TargetInfo::BranchProtectionInfo BPI(LangOpts);
BPI.setFnAttributes(FuncAttrs);
```